common/scala/src/main/resources/application.conf

```diff
@@ -12,3 +12,7 @@ akka.http {
         max-open-requests = 128
     }
 }
+
+#configs used by MesosContainerFactory
+#whisk.mesos.master-url = "http://192.168.99.100:5050"
```

Comment:
vs env var/current mechanism for threading parameters through (which is whisk config for now)?

Reply:
yes, currently there is no way to have an SPI impl advertise required properties to the whisk config; and we are already using typesafe config via akka

Reply:
with pure config, is this necessary?

Reply:
Can we move this into a nested structure like all other configuration as well? Is this needed to be commented out?